lib/livebook/application.ex

```diff
@@ -333,6 +339,21 @@ defmodule Livebook.Application do
     })
   end
 
+  defp create_online_hub(teams_key, name, org_id, org_key_id, agent_key) do
+    Livebook.Hubs.save_hub(%Livebook.Hubs.Team{
+      id: "team-#{name}",
+      hub_name: name,
+      hub_emoji: "💡",
+      user_id: nil,
```

Comment:
We should sync this with offline. Either both are zero or both are nil. If we are setting them to nil, we should also set org_key_id and org_id to nil.